internal/handlers/pg/msg_update.go

```diff
@@ -154,6 +158,10 @@ func (h *Handler) MsgUpdate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 			continue
 		}
 
+		if len(resDocs) > 1 && !multi {
+			resDocs = resDocs[0:1]
```

Comment:
I have a bad feeling about this because I don't understand if `resDocs[0]` is _always_ going to be the same document MongoDB would choose for an update, but I've checked out this branch and played with it a bit and it seems like Mongo and FerretDB are actually updating the same document, so it seems to work fine

Reply:
Thank you for the review! I agree with you and I also only have empirical evidence that it works. I will go ahead and add a FIXME note to the code along with the updates the feedback you provided above.